<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Refrensi">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    
	border-image: url(:/Main/Refrensi.png);
    border-radius: 10px;
    transition: all 0.2s ease-in-out;
}

QPushButton:hover {
    transform: scale(1.1);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Controller">
    <property name="geometry">
     <rect>
      <x>515</x>
      <y>300</y>
      <width>250</width>
      <height>143</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    
	border-image: url(:/Main/PID.png);
    border-radius: 10px;
    transition: all 0.2s ease-in-out;
}

QPushButton:hover {
    transform: scale(1.1);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Plant">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>300</y>
      <width>250</width>
      <height>143</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro Display</family>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    
	border-image: url(:/Main/Plant.png);
    border-radius: 10px;
    transition: all 0.2s ease-in-out;
}

QPushButton:hover {
    transform: scale(1.1);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="scopeOutput">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>320</y>
      <width>79</width>
      <height>100</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    
	border-image: url(:/Main/Scope.png);
    border-radius: 10px;
    transition: all 0.2s ease-in-out;
}

QPushButton:hover {
    transform: scale(1.1);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="scopeController">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>190</y>
      <width>79</width>
      <height>100</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    
	border-image: url(:/Main/Scope.png);
    border-radius: 10px;
    transition: all 0.2s ease-in-out;
}

QPushButton:hover {
    transform: scale(1.1);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Submit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>690</y>
      <width>700</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-image: url(:/Main/Submit Button.png);
    border-radius: 10px;
    transition: all 0.2s ease-in-out;
}

QPushButton:hover {
    transform: scale(1.1);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="runSim">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>690</y>
      <width>700</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    
	border-image: url(:/Main/Run Sim Button.png);
    border-radius: 10px;
    transition: all 0.2s ease-in-out;
}

QPushButton:hover {
    transform: scale(1.1);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1600</width>
      <height>900</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Main/Background Baru.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="scopeOutput_Diskrit">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>460</y>
      <width>79</width>
      <height>100</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    
	border-image: url(:/Main/Scope.png);
    border-radius: 10px;
    transition: all 0.2s ease-in-out;
}

QPushButton:hover {
    transform: scale(1.1);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="Overshoot">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="TimeS">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro Display</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Overshoot</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>181</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro Display</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Time Settling</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>Refrensi</zorder>
   <zorder>Controller</zorder>
   <zorder>Plant</zorder>
   <zorder>scopeOutput</zorder>
   <zorder>scopeController</zorder>
   <zorder>Submit</zorder>
   <zorder>runSim</zorder>
   <zorder>scopeOutput_Diskrit</zorder>
   <zorder>Overshoot</zorder>
   <zorder>TimeS</zorder>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../Asset/Resource.qrc"/>
 </resources>
 <connections/>
</ui>
